index.html

```diff
@@ -9,11 +9,66 @@
     <link rel="stylesheet" href="css/style.css">
     <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
 </head>
+
 <body>
+  <header>
+    <img class="img" src="./img/karma-logo.svg" alt="Logo"></img>
```

Comment:
Good to see usage of alt text here, it's great that you had accessibility in mind.